data/hai/hai reveal 5 philosophers.txt

```diff
@@ -212,22 +212,72 @@ mission "Hai Reveal [B03] Contact Giti"
-			`	Teeneep seems to bristle but Sayari smoothly cuts in. "You should not concern yourself with our comfort Admiral. It would not be seemly for the Elder to abandon the field while another leader remains." She looks pointedly at Choot'k. "You may have forgotten also but there are two heads of state, as you call them, in this room. It would also not be seemly for our ships to act in concert with the Unfettered without a proper diplomatic liaison here. And so I too have my place here. Things have happened very quickly of course, but my colleague Hiyamaharu will soon be arriving at Mountaintop to act as my counterpart there. I trust things are being attended to, but you might consider checking that your people are properly aware of that."`
-			`	Danforth gives his lieutenant a 'follow that up' look and then looks around. "Right, well in that case, the other officers should all be here and we can review our plans for stage two in an hour. <last>, thank you for your recent efforts, but you won't be invited to that one. Meeting adjourned in favor of coffee."`
+			`	Teeneep seems to bristle but Sayari smoothly cuts in. "You should not concern yourself with our comfort Admiral. It would not be seemly for the Elder to abandon the field while another leader remains." She looks pointedly at Choot'k. "You may have forgotten also but there are Hai representatives of two governments in this room. It would be unseemly for our ships to act in concert with the Unfettered without a diplomatic liaison here. And so I too have my place here. Things have happened quickly of course, but my colleague Hiyamaharu will soon arrive at Mountaintop to act as my counterpart there. I trust things are being attended to, but you might consider checking that your people are properly aware of that."`
+				goto briefing
+
+			label offended
+			`	Everyone looks surprised or offended, as if you are the court jester who thought he was king. They're dismissive of your comment, as they begin to discuss your departure.`
+				goto done
+			label honored
+			`	The leaders all look pleased. General Choot'k says, "You've surpassed my expectations, Captain <last>. I've never seen a fleet so large, but you led it to victory with the speed and skill of a veteran general." Danforth looks just as impressed.`
+			label done
+			action
+			`	Danforth explains, "For now, all communication in and out of the area is blocked at the relay unless its to or from specific high-level Navy or Syndicate sources. Captain <last>, I'm afraid that also includes you. We don't have time or equipment to harden your systems against potential intrusion, but you do have the nearly exclusive freedom to pass through the blockade. You're a pilot, not a ground combat troop operator, so you should pursue other matters."`
 			`	On your way out Sayari catches you and suggests meeting aboard Teeneep's ship to discuss some things.`
```

Comment:
```suggestion
			`	On your way out Sayari catches you and suggests meeting aboard Teeneep's ship to discuss the kidnappings.`
```
Anything is better than 'some things'. Maybe there's still a better word than 'kidnapping' though.

Reply:
They Hai and Captain Last are trying to keep the cadmium poisoning secret (not sure why) so she's being intentionally vague. I'll update this, so she actually says something vague.

Reply:
Now Sayari actually speaks, and there's a choice that breaks up the conversation, so there's a scene transition.